camp/nfutsum/nfutsum-miniproject/XBox Controller Left and Right bumper/src/main/java/frc/robot/commands/MotorBackwardCommand.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.MotorSubsystem;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+
+/** An example command that uses an example subsystem. */
+public class MotorBackwardCommand extends CommandBase {
+  private final MotorSubsystem theSubsystem;
+
+  /**
+   * Constructer that takes the motor subsystem
+   *
+   * @param m_motor The motor subsystem
+   */
+  public MotorBackwardCommand(MotorSubsystem m_motor) {
+    theSubsystem = m_motor;
```

Comment:
Variable naming starting with 'm_' for when it is being passed as a parameter is incorrect.  Sure it will compile, but m_ usually means class member.  When a class is passed by reference, that inherently means it is not a member of the class.  Also, as you may have seen previously, m_ naming is not really something we should practice here.  Our north star is standard Java naming conventions, namely camel casing (other links that we should try to follow include: https://www.javatpoint.com/java-naming-conventions and/or https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)